config/crd/patches/cainjection_in_clusterbootstraps.yaml

```diff
@@ -0,0 +1,8 @@
+# The following patch adds a directive for certmanager to inject CA into the CRD
```

Comment:
Are we adding the cainjection files back. I also had added them but had to remove them. So we can be consistent here for all the controllers

Reply:
removed